src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -1156,7 +1162,7 @@ namespace QgsWfs
             query.removeAllQueryItems( param.first );
         }
 
-        query.addQueryItem( QStringLiteral( "REQUEST" ), QStringLiteral( "DescribeFeatureType" ) );
+        query.addQueryItem( QStringLiteral( "REQUEST" ), QStringLiteral( "GetFeature" ) );
```

Comment:
this change is not correct. The xsi:schemaLocation of a GetFeature response should point to a DescribeFeatureType request to get the schema